Makefile.am

```diff
@@ -7,6 +7,16 @@ MAINTAINERCLEANFILES = test-driver \
 	config.sub depcomp install-sh ltmain.sh m4/* missing \
 	depcomp ylwrap
 
+if AIX
+SHLIBEXT=a
+LNFLAG=fs
+KILLALL = ./kill-all
```

Comment:
Is there a reason why you add the kill-all script only later in commit "tools: Add missing code that is AIX-specific" ? Why not adding it with this commit already ?

Reply:
I wanted to bunch up all new file additions into one commit, so they could all be modified together. I tried making each commit compilable by itself, but for some it wasn't possible without also mixing different code areas into the same commit.

Reply:
OK, fine with me.